code/game/objects/items/mountable_frames/frames.dm

```diff
@@ -25,3 +25,16 @@
 			user << "<span class='rose'>[src] cannot be placed in this area.</span>"
 			return
 		return 1
+
+/obj/item/mounted/frame/newscaster
```

Comment:
To be honest, I hate the fact that all the mounted frames are in their own files when they are so short. It should be the opposite, in my mind: let's organize them like we do vending machines: all in the parent file.

If it were up to me, I would put all the other frames into frames.dm